mautic/mautic
# [DPMMA-507] Add permission export audit and history lead

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🟢 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description
The change also adds the user's history and AuditLog to the contact export permission. A bug related to a validation error and throwing error 500 in the case of empty log results has also been fixed.

![image](https://github.com/user-attachments/assets/a49ff1ca-4bd1-44ce-9bb4-e46539ba13d5)







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. change the user's permissions so that he cannot export the contact
3. go to the website https://mautic5.ddev.site/s/contacts/view/{ID}, you will not be able to export data
